Task9/Task9.java

```diff
@@ -0,0 +1,39 @@
+
+import java.util.Scanner;
+
+public class Task9 {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter an number from 0 to 100: ");
+        int userInput = scanner.nextInt();
+
+        if (userInput < 0 || userInput > 100)
+        {
```

Comment:
Java formating must be:
if (userInput < 0 || userInput > 100) {
"{" should be on the same line